docs/user/courses/customizable.rst

```diff
@@ -0,0 +1,75 @@
+Customizing Courses
+=================
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Courses contain both exercises and exams and different roles can be attached to different sets of course participants: Instructors, Editors, Tutors and Users.
+You can find a detailed listing of the different roles and their corresponding permissions `here </admin/accessRights>`_.
+
+Creating courses
+----------------
+
+Administrators can create new courses by clicking on |course_management| in the header and then clicking on |course_create|.
+
+Administrators can then specify options in the creation form, instructors can later change these values:
+
+- **Title**: Name of the course that is shown to users e.g. in the |course_overview|. Can be changed later.
+- **Short Name**: Unique identifier of the course. Can **not** be changed later as it is included in resources of external systems such as build plans.
+- **Color**: Color of the course that will be used e.g. in the |course_overview|.
+- **Course Description**: Description of the course that will e.g. be shown in the Course Registration table and in the header of the course (once a user clicks on the course in their dashboard).
+- **Course Organizations**: Users and Courses can be grouped into different organizations (e.g. based on the email-address of users). Users will then only see courses from their organizations, allowing multiple organizations to use the same Artemis environment without interfering with each other.
+  If no course organization is set, all users can interact with the course.
+- **Start/End**: Start and End Date of the course. Users can only self-register to a course between the Start and End date.
```

Comment:
You capitalized ``Date`` at the beginning of the sentence, so here as well I guess?
```suggestion
- **Start/End**: Start and End Date of the course. Users can only self-register to a course between the Start and End Date.
```